src/main/java/com/EighthLight/app/Board.java

```diff
@@ -127,17 +126,20 @@ public void setWinningCombos() {
     }
 
     public boolean hasAPlayerWon(ArrayList boardState, String playerSymbol) {
```

Comment:
I was having trouble understanding what this did, so I broke it up into two methods for better clarity.